.changeset/tabaxi-fellas-inspire.md

```diff
@@ -0,0 +1,147 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"__section": tree
+---
+TableSchema (alpha) APIs have been added
+
+A `TableSchema` utility has been added to Shared Tree for managing dynamic, tabular data.
+This new `TableSchema` namespace contains APIs for creating column, row, and table [node schema](https://fluidframework.com/docs/api/fluid-framework/treenodeschema-typealias).
+
+Note: these APIs require the use of [SchemaFactoryAlpha](https://fluidframework.com/docs/api/fluid-framework/schemafactoryalpha-class).
+
+> [!WARNING]
+> These APIs are in preview and are subject to change.
+> Until these APIs have stabilized, we do not recommend using them in production code.
+> We reserve the right to make breaking changes to these APIs, including their persisted data format.
+> Using these APIs in production code may result in data loss or corruption.
+
+#### Creating a table with default column and row schema
+
+You can craft a table with defaults via `TableSchema.createTable`, without specifying custom column or row schema.
+Note that you will still be required to provide a schema for the cells that will appear in the table.
+
+```typescript
+class Cell extends schemaFactory.object("TableCell", {
+	value: schemaFactory.string,
+}) {}
+
+class Table extends TableSchema.createTable({
+	schemaFactory,
+	cell: Cell,
+}) {}
+
+const table = new Table({
+	columns: [{ id: "column-0" }],
```

Comment:
I think it will be common for users to want to declare an unhydrated column node in its own variable so they can then reference it in the table and use it manuliplating cells. We might want to show that pattern somewhere, which could be here.